src/main/java/com/mastercard/developer/encryption/JsonParser.java

```diff
@@ -86,4 +86,16 @@ static Object readJsonObject(DocumentContext context, String jsonPathString) {
         }
         return jsonElement;
     }
+
+    // Upgrading the json-path lib from 2.6.0 to 2.9.0 introduced a bug where when you
```

Comment:
please remove this comment

Reply:
Sorry, I meant : remove all of this:     // try to delete a non-existent key in a DocumentContext with the SUPPRESS_EXCEPTIONS flag,
    // it would throw a ClassCastException. This method is a workaround for the issue.
    // Once this issue is fixed this method's usages can be replaced with a simple DocumentContext.delete(path).
    // Track the issue here https://github.com/json-path/JsonPath/issues/870

Reply:
We could leave this comment in until this issue is fixed. It's a good way to make sure we revisit it in the future.

Reply:
Sorry, folks, I would insist that we remove this comment. There will be no reason to update this library, unless another security issue. let keep it code clean.

Reply:
removed it now.